Scripts/SearchIncidents/SearchIncidents.py

```
@@ -0,0 +1,60 @@
+from typing import Dict, List
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+
+special = ['n', 't', '\\', '"', '\'', '7', 'r']
+
+
+def check_if_found_incident(res: List):
+    if res and isinstance(res, list) and isinstance(res[0].get('Contents'), dict):
+        if 'data' not in res[0]['Contents']:
+            raise DemistoException(res[0].get('Contents'))
+        elif res[0]['Contents']['data'] is None:
+            raise DemistoException("Incidents not found.")
+
+    else:
+        raise DemistoException(f'failed to get incidents from demisto.\nGot: {res}')
+
+
+def is_valid_args(args: Dict):
+    error_msg: List[str] = []
+    for _key, value in args.items():
+        i = 0
+        while i < len(value):
+            if value[i] == '\\':
+                if value[i + 1] not in special:
+                    error_msg.append(f'Error while parsing the argument: "{_key}" '
+                                     f'\nSucceeded parsing untill:\n- "{value[0:i]}"')
+                else:
+                    i += 1
+            i += 1
+    if len(error_msg) != 0:
+        return_error('\n'.join(error_msg))
```

raise an exception instead of return_error